plugin/lighthouse/handler_test.go

```diff
@@ -43,9 +52,12 @@ func testWithoutFallback() {
 	)
 
 	BeforeEach(func() {
+		mockCs := new(MockClusterStatus)
+		mockCs.clusterId = clusterID
```

Comment:
We should add a case where `IsConnected` returns false. Also a case where there's 2 clusters and one returns false and the other true.

Reply:
done